deriva/core/ermrest_model.py

```diff
@@ -19,6 +19,23 @@ class Model (_ec.CatalogConfig):
     """
     def __init__(self, model_doc, **kwargs):
         super(Model, self).__init__(model_doc, **_kwargs(**kwargs))
+        Model.update_referenced_by(self)
+
+    @classmethod
```

Comment:
I don't understand why this is a classmethod.  It should be a regular method taking `self` which is the model being updated.